docs/source/developer/django_setup.rst

```diff
@@ -42,4 +42,22 @@ applications within the database.
 The rest of the :doc:`index` will inform you of how to develop the
 applications and other components of the CS Unplugged project.
 
+Loaders
+==============================================================================
+
+To populate the database with content we have written a series of custom loaders.
+The loaders for an application are found in the ``management/commands/`` directory, and
+there is approximately one loader for each configuration file.
+
+Besides just populate fields in the database, a loader is also responsible for checking
```

Comment:
Start of sentence doesn't really make sense, should be 'Besides populating fields...'